docs/UserGuide.md

```diff
@@ -166,7 +166,8 @@ This section aims to provide you with in-depth details on Fine\$\$e's unique fea
 **:warning: &nbsp; IMPORTANT &nbsp; :warning:**<br>
 
 The formats of the parameters used in the rest of the document are as follows:
-* `TITLE` and `CATEGORY` should consist of <abbr title="Alphanumeric characters, space, and the special characters !&quot;#$%&'()*+,-./:;&lt;=&gt;?@[\]^_`{\|}~">printable ASCII characters</abbr>, and cannot begin with a space.
+* `TITLE` and `CATEGORY` should consist of printable ASCII characters, and cannot begin with a space.
+  * Printable ASCII characters consist of alphanumeric characters `0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz`, space, and the special characters ``!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~``.
```

Comment:
It's slightly odd that the space character is the only character not explicitly represented. However, I don't see a good way to express it on its own unless all three parts are combined i.e.
```
0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
```